File: Actuarialvaluations.Models/API/FileUpload.cs
namespace Actuarialvaluations.Models.API
{
  /// <summary>
  /// Gets or sets Applicable Model Point Variable
  /// </summary>
  public class FileUpload
  {
    /// <summary>
    /// Gets or sets File Classification
    /// </summary>
    public string Classification { get; set; }

    /// <summary>
    /// Gets or sets Table Name
    /// </summary>
    public string TableName { get; set; }

    /// <summary>
    /// Gets or sets Location Type
    /// </summary>
    public string LocationType { get; set; }

    /// <summary>
    /// Gets or sets File Type
    /// </summary>
    public string Type { get; set; }

    /// <summary>
    /// Gets or sets File Path
    /// </summary>
    public string Path { get; set; }
  }
}

File: Actuarialvaluations/clientapp/src/models/entities/Product/FileUpload.ts
/**
 * The File Upload model class
 */
export class FileUpload {
    public Classification: string = '';
    public TableName: string = "";
    public LocationType: string = "";
    public Type: string = "";
    public Path: string = "";
    
    constructor(data: any = null) {
        if (data) {
            Object.assign(this, data);
        }
    }
}